content/master/getting-started/provider-azure-part-2.md

```diff
@@ -401,15 +401,6 @@ spec:
                   matchControllerRef: true
                 resourceGroupNameSelector:
                   matchControllerRef: true
-          patches:
```

Comment:
ah interesting, it does look like `Subnet` is the only resource in this composition that doesn't have a `spec.forProvider.location` field. Nice catch!

https://marketplace.upbound.io/providers/upbound/provider-azure-network/v1.5.0/resources/network.azure.upbound.io/Subnet/v1beta1